R/ipr2viz.R

```diff
@@ -152,35 +152,35 @@ ipr2viz <- function(infile_ipr = NULL, infile_full = NULL, accessions = c(),
     # Filter by Top Accessions per Accession per DomArch and Lineage
     ipr_out <- subset(
         ipr_out,
-        ipr_out$AccNum %in% top_acc
+        ipr_out$.data$AccNum %in% top_acc
     )
     ## Need to fix this eventually based on the 'real' gene orientation! :)
     ipr_out$Strand <- rep("forward", nrow(ipr_out))
 
-    ipr_out <- ipr_out %>% arrange(AccNum, StartLoc, StopLoc)
+    ipr_out <- ipr_out %>% arrange(.data$AccNum, .data$StartLoc, .data$StopLoc)
     ipr_out_sub <- filter(
         ipr_out,
-        grepl(pattern = analysis, x = Analysis)
+        grepl(pattern = analysis, x = .data$Analysis)
     )
     # dynamic analysis labeller
     analyses <- ipr_out_sub %>%
-        select(Analysis) %>%
+        select(.data$Analysis) %>%
         distinct()
     analysis_labeler <- analyses %>%
-        pivot_wider(names_from = Analysis, values_from = Analysis)
+        pivot_wider(names_from = .data$Analysis, values_from = .data$Analysis)
 
     lookup_tbl_path <- "/data/research/jravilab/common_data/cln_lookup_tbl.tsv"
-    lookup_tbl <- read_tsv(lookup_tbl_path, col_names = T, col_types = lookup_table_cols)
+    lookup_tbl <- read_tsv(lookup_tbl_path, col_names = T, col_types = .data$lookup_table_cols)
 
-    lookup_tbl <- lookup_tbl %>% select(-ShortName) # Already has ShortName -- Just needs SignDesc
-    # ipr_out_sub = ipr_out_sub %>% select(-ShortName)
+    lookup_tbl <- lookup_tbl %>% select(-.data$ShortName) # Already has ShortName -- Just needs SignDesc
+    # ipr_out_sub = ipr_out_sub %>% select(-.data$ShortName)
     # TODO: Fix lookup table and uncomment below
     # ipr_out_sub <- merge(ipr_out_sub, lookup_tbl, by.x = "DB.ID", by.y = "DB.ID")
 
     ## PLOTTING
     ## domains as separate arrows
     # For odering with tree
-    # ipr_out_sub$Name <- paste0(" ", ipr_out_sub$Name)
+    # ipr_out_sub$.data$Name <- paste0(" ", ipr_out_sub$.data$Name)
```

Comment:
.data is not needed here, as `ipr_out_sub$Name` was explicit about where the variable came from.